shell/src-app/Linter.hs

```diff
@@ -0,0 +1,83 @@
+-- Allow ourselves to write Text literals.
+{-# LANGUAGE OverloadedStrings #-}
+
+module Main where
+
+    import Prologue hiding (Symbol)
+    import Luna.Syntax.Text.Lexer.Runner
+    import Luna.Syntax.Text.Lexer.Token as T
+    import Luna.Syntax.Text.Lexer.Symbol as S
+    import Data.Text.Position            (Delta)
+    import Data.Text32 hiding (convert)
+    import Data.Aeson
+    import qualified Data.ByteString.Lazy as B
+
+    instance ToJSON Delta where
+        toJSON delta = toJSON ((convert delta) :: Int)
+
+    jsonStr :: String -> Value
+    jsonStr = toJSON
+
+    jsonT32 :: Text32 -> Value
+    jsonT32 txt = toJSON ((convert txt) ::String)
+
+    symbolWithValue :: String -> Text32 -> Value
+    symbolWithValue name value = object 
+        [ "symbol" .= toJSON name
+        , "value" .= jsonT32 value ]
```

Comment:
wrong code formatting, the closing bracket goes to the next line. See our code formatting guide and apply to other parts as well.